lib/assertions/directory_content.js

```diff
@@ -0,0 +1,213 @@
+module.exports = function (chai, utils) {
+
+	var Assertion = chai.Assertion;
+	var AssertionError = chai.AssertionError;
+	var flag = utils.flag;
+	var assert = chai.assert;
+
+	var readdir = require('readdir-enhanced');
+	var format = require('util').format;
```

Comment:
fyi chai has its own message formatting built in. Have a look at some of the code codebase to see how it can work.